presto-hive/src/test/java/com/facebook/presto/hive/TestHiveFileFormats.java

```diff
@@ -202,7 +202,7 @@ public void testParquetPageSource(int rowCount)
                 .filter(column -> column.getObjectInspector().getCategory() == Category.PRIMITIVE)
                 .collect(toList());
         TestingConnectorSession session = new TestingConnectorSession(
-                new HiveSessionProperties(new HiveClientConfig().setParquetOptimizedReaderEnabled(true)).getSessionProperties());
+                new HiveSessionProperties(new HiveClientConfig().setParquetOptimizedReaderEnabled(true).setParquetPredicatePushdownEnabled(true)).getSessionProperties());
```

Comment:
From what I understood, this particular test seemed to be the test for the optimized reader, so it made sense to add testing predicate pushdown here too. Since predicate pushdown isn't the default in the product, we should have some tests that don't use it.